karma.conf.js

```diff
@@ -283,7 +287,7 @@ module.exports = function (config) {
 			jsx: "preserve",
 
 			// esbuild options
-			target: downlevel ? "es5" : "es2015",
+			target: downlevel ? "es5" : "es2019",
```

Comment:
Whoops - will undo this. Did this to not downlevel async/await while debugging tests